README.md

```diff
@@ -197,3 +197,66 @@ func main() {
     // ...
 }
 ```
+
+### Redis
+
+The [github.com/segmentio/stats/redisstats](https://godoc.org/github.com/segmentio/stats/redisstats)
+package exposes:
+
+* a decorator of
+  [`redis.RoundTripper`](https://godoc.org/github.com/segmentio/redis-go#RoundTripper)
+  which collects metrics for client requests, and
+* a decorator or
+  [`redis.ServeRedis`](https://godoc.org/github.com/segmentio/redis-go#HandlerFunc.ServeRedis)
+  which collects metrics for server requests.
+
+Here's an exmaple of how to use the decorator on the client side:
+```go
+package main
+
+import (
+    "github.com/segmentio/redis-go"
+    "github.com/segmentio/stats/redisstats"
+)
+
+func main() {
+    stats.Register(datadog.NewClient("localhost:8125"))
+    defer stats.Flush()
+
+    client := redis.Client{
+        Addr:      "127.0.0.1:6379",
+        Transport: redisstats.NewTransport(&redis.Transport{}),
+        Timeout:   0,
+    }
+
+    // ...
+}
+```
+
+And on the server side:
+
+```go
+package main
+
+import (
+    "github.com/segmentio/redis-go"
+    "github.com/segmentio/stats/redisstats"
+)
+
+func main() {
+    stats.Register(datadog.NewClient("localhost:8125"))
+    defer stats.Flush()
+
+    handler := redis.Handler{}
```

Comment:
Sorry I missed that earlier, here we should replace with something like this:
```go
handler := redis.HandlerFunc(func(res redis.ResponseWriter, req *redis.Request) {
  // ...
})
```
and bellow
```go
server := redis.Server{
    Handler: redisstats.NewHandler(handler),
}
```
`redis.Handler` is an interface and therefore cannot be instantiated.